client/components/keyed-suggestions/index.jsx

```diff
@@ -18,12 +18,14 @@ import {
 } from 'lodash';
 import classNames from 'classnames';
 import i18n from 'i18n-calypso';
+import { cosineSimilarity } from 'calypso/lib/trigram';
```

Comment:
Nit: This import is an internal one so it should be under the `Internal dependencies` docblock (which we seem to have to add to this file).